commands.h

```diff
@@ -44,6 +44,7 @@ bool ct_scroll(arg_t);
 bool ct_drag_mark_image(arg_t);
 bool ct_select(arg_t);
 
+#ifdef _MAPPINGS_CONFIG
```

Comment:
or maybe it's better to declare all these functions in `nsxiv.h` as well?
Hmmm... or maybe not. They're only be relevant for keybindings.